`make archive` fails on macOS

## Issue type ##

Bug Report

## Build number #

master

## Summary ##

`make archive` fails on macOS with the following error:

> sed: 1: "build/s3/custom/templat ...": undefined label 'uild/s3/custom/template_build_9b438dc.yaml'
make: *** [archive] Error 1

## Steps to reproduce ##

Run `make archive` on macOS.

## Expected results ##

It should work.

## Actual results ##

![image](https://user-images.githubusercontent.com/610129/73126256-6ddf4780-3f7e-11ea-9567-bf743087f3e8.png)

--- a/Makefile
+++ b/Makefile
@@ -55,7 +55,8 @@ archive: build                                               ## Create archive t
 	@cp -f cloudformation/stacks/AutoSpotting/template.yaml $(LOCAL_PATH)/template_build_$(BUILD).yaml
 	@zip -j $(LOCAL_PATH)/regional_stack_lambda.zip cloudformation/stacks/AutoSpotting/regional_stack_lambda.py
 	@cp -f cloudformation/stacks/AutoSpotting/regional_template.yaml $(LOCAL_PATH)/
-	@sed -i "s#lambda\.zip#lambda_build_$(BUILD).zip#" $(LOCAL_PATH)/template_build_$(BUILD).yaml
+	@sed -e "s#lambda\.zip#lambda_build_$(BUILD).zip#" $(LOCAL_PATH)/template_build_$(BUILD).yaml > $(LOCAL_PATH)/template_build_$(BUILD).yaml.new
+	@mv -- $(LOCAL_PATH)/template_build_$(BUILD).yaml.new $(LOCAL_PATH)/template_build_$(BUILD).yaml
 	@cp -f $(LOCAL_PATH)/lambda.zip $(LOCAL_PATH)/lambda_build_$(BUILD).zip
 	@cp -f $(LOCAL_PATH)/lambda.zip $(LOCAL_PATH)/lambda_build_$(SHA).zip
 	@cp -f $(LOCAL_PATH)/regional_stack_lambda.zip $(LOCAL_PATH)/regional_stack_lambda_build_$(BUILD).zip

